tests/main/snap-auto-import-asserts-spools/task.yaml

```diff
@@ -36,17 +37,17 @@ execute: |
     echo "Simulate a not running snapd (happens on e.g. early boot)"
     systemctl stop snapd.service snapd.socket
 
-    echo "`snap auto-import` spooled assertions if it can not talk to snapd"
+    echo "'snap auto-import' spooled assertions if it can not talk to snapd"
     snap auto-import
     ls /run/snapd/auto-import
     umount /mnt
     systemctl start snapd.service snapd.socket
 
-    echo "`snap auto-import` reads from the auto-import dir"
+    echo "'snap auto-import' reads from the auto-import dir"
     snap auto-import
     snap known account-key | MATCH "name: test-store"
 
-    nr=$(ls /run/snapd/auto-import|wc -l)
+    nr=$(find /run/snapd/auto-import -maxdepth 1 -mindepth 1 |wc -l)
```

Comment:
Curious, why do we need mindepth 1 here?

Reply:
That's because:
```
$ mkdir foo
$ ls foo
$ find foo -maxdepth 1 -mindepth 1
$ find foo -maxdepth 1
foo
```
Either that or the `if` below would need `"$nr" -gt 1`